bentoml/adapters/multi_image_input.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Atalaya Tech, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from bentoml.adapters.base_input import BaseInputAdapter
+
+
+class MultiImageInput(BaseInputAdapter):
+    """
+    Multiple images input adapter
```

Comment:
Could you also add example usage and init parameters here?  

You should also add it to `bentoml.adapters` and the docs' adapters list page, just make sure the docstring here starts with something like "Warning: this is a not yet implemented, follow issue #XYZ for updates", you can create a Github issue for it and use the issue id in the comment